sysinit/dune

```diff
@@ -5,7 +5,7 @@
  (modules coqargs)
  (wrapped false)
  ; don't depend on rocq-runtime.lib -> impossible to imperatively set random flags
- (libraries rocq-runtime.config rocq-runtime.clib))
+ (libraries rocq-runtime.config rocq-runtime.boot rocq-runtime.clib))
```

Comment:
Why this change?

Reply:
to have access to the `query` type

Reply:
Oh indeed, this is `clib` not `lib`

Reply:
Not a big deal, but for the future we could move `usage.ml` to sysinit and do a bit more refactoring. In master right now usage.ml is orphan there, I guess due to rocqshim changes.